fedmsg_genacls.py

```diff
@@ -98,7 +98,8 @@ def delayed_consume():
 
         self.queued_messages.append(msg)
 
-        moksha.hub.reactor.reactor.callLater(self.delay, delayed_consume)
+        moksha.hub.reactor.reactor.callFromThread(
+            moksha.hub.reactor.reactor.callLater, self.delay, delayed_consume)
```

Comment:
@ralphbean correct your link not to link to branch, but tag or commit to be permalink